src/System.ComponentModel.TypeConverter/src/System/ComponentModel/CultureInfoConverter.cs

```diff
@@ -28,7 +28,7 @@ public class CultureInfoConverter : TypeConverter
         /// <summary>
         /// Retrieves the Name for a input CultureInfo.
         /// </summary>
-        protected virtual string GetCultureName(CultureInfo culture) => culture.Name;
+        protected virtual string GetCultureName(CultureInfo culture) => culture?.Name;
```

Comment:
Opted to return null instead of throwing ANE

Reply:
@maryamariyan this is the fix for the NRE bug. I think it is worth fixing. We don’t consider this breaking (the guidelines say exception -> no exception is not breaking)